WordPress/gutenberg
# Add `wp-env` After Setup Command

## What?
This pull request will execute a new lifecycle script, `afterSetup`, whenever WordPress is configured. It will run on:

- `wp-env start`: Runs when the cache is invalidated and WordPress is configured.
- `wp-env clean`: Runs after the environment is cleaned.

## Why?
By allowing users to run their own command after installation, they will be able to define setup scripts and the like that perform additional processing on the environment post-setup.

## How?
We've added an `afterSetup` option that can be set at the root of a `.wp-env.json` file. This option can also be overridden using the `WP_ENV_AFTER_SETUP` environment variable.

## Testing Instructions

1. Create (or edit) a `.wp-env.override.json` to include:
```json
{
  "afterSetup": "echo \"Test\""
}
```
2. Run `npm run env -- start --update` and verify that `Test` is printed once.
3. Run `npm run env -- start --update --no-execute-scripts` and verify that `Test` is not printed.
4. Run `WP_ENV_AFTER_SETUP="ls -la" npm run env -- start --update` and verify that the Gutenberg directory is printed once.
5. Run `npm run env -- clean` and verify that `Test` is printed once.
6. Run `npm run env -- clean  --no-execute-scripts` and verify that `Test` is not printed.
